app/controllers/homepage_controller.rb

```diff
@@ -156,30 +155,30 @@ def find_listings(params, listings_per_page, filter_params, includes, location_s
     dropdowns = filter_params[:custom_dropdown_field_options].map { |dropdown_field| dropdown_field.merge(type: :selection_group, operator: :or) }
     numbers = numeric_search_params.map { |numeric| numeric.merge(type: :numeric_range) }
 
-    coordinates = Maybe(params[:lc]).map { search_coordinates(params[:lc]) }.or_else({})
     distance_unit = (location_search_in_use && MarketplaceService::API::Api.configurations.get(community_id: @current_community.id).data[:distance_unit] == :metric) ? :km : :miles
-
-    distance_max = [5, params[:distance_max]].max if params[:distance_max].present?
+    location_search_params = location_search_params(
```

Comment:
How would you feel about renaming this locale variable. My worry is that it is easy to confuse the variable name with the method name now.